app/MMB/src/context/__tests__/BookingDetailContext.test.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import * as React from 'react';
+import renderer from 'react-test-renderer';
+
+import BookingDetailContext from '../BookingDetailContext';
+
+class ContextConsumer extends React.Component<{}> {
+  render = () => null;
+}
+
+describe('BookingDetailContext', () => {
+  it('sets the isPastBooking', () => {
+    const wrapper = renderer.create(
+      <BookingDetailContext.Provider>
+        <BookingDetailContext.Consumer>
+          {({ isPastBooking, actions: { setIsPastBooking } }) => (
+            <ContextConsumer
+              isPastBooking={isPastBooking}
+              setIsPastBooking={setIsPastBooking}
+            />
+          )}
+        </BookingDetailContext.Consumer>
+      </BookingDetailContext.Provider>,
+    );
+
+    const instance = wrapper.root.findByType(ContextConsumer);
+    instance.props.setIsPastBooking(true);
+
+    expect(instance.props.isPastBooking).toBe(true);
```

Comment:
Nice test 👍 - one small detail - maybe try to test even the `false` value right after this because this assumes that the default value is `false` (this test will be pointless if someone changes it to `true`).

Reply:
Maybe test that it is false before I call set state?

Reply:
Yes, that's also an option.

Reply:
Implemented and pushed